make/ingress/run

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+if [ -z "${INGRESS_CONTROLLER}" ]; then
```

Comment:
Is this always set? Do we need to use `if [ -z "${INGRESS_CONTROLLER:-}" ]` instead? (I guess we don't have `-o nounset` here… which then brings up the question, why don't we?)

Reply:
The script was simple enough for not using `-o nounset`, but I guess it is just better having it then? Pushing on the next commit.